teams.json

```diff
@@ -17,20 +17,40 @@
       "name": "code-build-run",
       "apis": [
         "appengine",
+        "artifactregistry",
         "cloudbuild",
         "cloudfunctions",
         "cloudscheduler",
         "cloudtasks",
         "container",
-        "containeranalysis",
         "run",
         "logging",
-        "clouderrorreporting"
+        "clouderrorreporting",
+        "containeranalysis",
+        "containerregistry"
       ],
       "repos": [
         "googleapis/nodejs-logging-winston",
         "googleapis/nodejs-logging-bunyan",
+        "googleapis/nodejs-cloudbuild",
```

Comment:
For split repos bound to a given API (nodejs-cloudbuild, python-containeranalysis, etc) there is no need to explicitly list them here.  We get assignment based on the API labels that are applied automatically.  You can delete lines 35-40, and 42.